### PR TITLE
[es-AR] Updates to firefox/new.lang

### PR DESCRIPTION
**Pull request by gaby2300**
I thought there were 2 missing strings so I translated them and I edited the last one.It seems I was wrong though, I ended editing 3 strings after all, sorry!

### CODE REVIEW

File: es-AR/firefox/new.lang [hunk 1]
```diff
@@ -207,6 +207,6 @@ Dale a Firefox una puesta a punto
 # You should keep the translation consistent with Firefox:
 # http://transvision.mozfr.org/string/?entity=browser/chrome/browser/safeMode.dtd:refreshProfile.label&repo=beta
 ;Refresh Firefox
-Refrescar Firefox
+Actualizá Firefox
```
Comment (flodolo) [inline es-AR/firefox/new.lang]: Note that, at this point, the page is inconsistent with the UI actually shipping in Firefox for es-AR

**Comment by deimidis** [comment]
Thanks, flod, I update the file to match the UI

**Comment by gaby2300** [comment]
Yes, sorry! I mistook the meaning of "Refresh" in this context.
Deimidis, thanks for making it right!

**Comment by gaby2300** [comment]
Yes, sorry! I mistook the meaning of "Refresh" in this context.
Deimidis, thanks for making it right!

**Pull request merged**